internal/db/query.go

```diff
@@ -23,8 +23,10 @@ var (
 	ErrNotFound = errors.New("resource not found")
 )
 
-type EncodedContent = []byte
-type DecodedContent = []byte
+type (
+	EncodedContent = []byte
+	DecodedContent = []byte
+)
```

Comment:
These kinds of changes are from a more strict go formatter setting I have set somewhere... I can revert if you'd like but I thought it's okay to keep.